### PR TITLE
Feature/ranking service

### PR DESCRIPTION
**Pull request by mazanec202**

### CODE REVIEW

File: service/src/main/java/tennisclub/service/RankingServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import java.util.List;
+
+@Service
+public class RankingServiceImpl implements RankingService {
+
+    private final RankingDao rankingDao;
+
+    @Autowired
+    RankingServiceImpl(RankingDao rankingDao) { this.rankingDao = rankingDao; }
```
Comment (Fuyunori) [inline service/src/main/java/tennisclub/service/RankingServiceImpl.java:18]: I think this constructor should be `public`, not `package` private 🤔

File: service/src/main/java/tennisclub/service/RankingServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        return rankingDao.findByUser(player);
+    }
+
+    @Override
+    public List<Ranking> findByTournament(Tournament tournament) {
+        return rankingDao.findByTournament(tournament);
+    }
+
+    @Override
+    public void update(Ranking ranking) {
```
Comment (Fuyunori) [inline service/src/main/java/tennisclub/service/RankingServiceImpl.java:41]: I think the update method should return the updated object, however I am not sure about this myself 😄

**Review by Fuyunori — APPROVED** [review]
Apart from the comments, I think it is great! Thank you and good job❗ 👍

File: service/src/main/java/tennisclub/service/RankingServiceImpl.java (hunk 1 above)
Comment (mazanec202) [reply service/src/main/java/tennisclub/service/RankingServiceImpl.java:18]: Good spot, thanks! That's the result of not writing the tests myself and a lack of perfectionism 😄

File: service/src/main/java/tennisclub/service/RankingServiceImpl.java (hunk 2 above)
Comment (mazanec202) [reply service/src/main/java/tennisclub/service/RankingServiceImpl.java:41]: Makes sense as the DAO update returns a new entity, thanks! I'll have to fix this for the user as well 😄

**Pull request merged**